examples/AlwaysFails.purs

```diff
@@ -0,0 +1,136 @@
+-- | Warning: This contract will permanently lock 5ada
+-- | This module demonstrates how the `Contract` interface can be used to build,
+-- | balance, and submit a failing smart-contract transaction. It creates a
+-- | transaction that pays two Ada to the `AlwaysFails` script address, and
+-- | then attempts to spend the two Ada, failing and losing the collateral.
```

Comment:
Let's rename this example to LoseCollateral (it would be annoying to actually lose it by accidentally opening the example)

Reply:
Yeah I didn't include it in ByUrl for this reason. Also, I just realised it actually locks 7 ada. I'll update that too